### DISCUSSION TITLE
Why timeout didn't work?

### DISCUSSION THREAD

**Discussion by Clared1** [cypress-io/cypress#19391] [General]
![image](https://user-images.githubusercontent.com/47734806/146351193-6cfa105b-e7fb-42cb-9470-a39421fdd9ed.png)

**Comment by vaclavGabriel** [comment]
The 10000ms timeout is set up for the `cy.contains` command, not `cy.click`. Secondly, the command failed because the element is not visible. To fix it, either select a visible command or add `{force:true}` to the cy.click command.
